src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29661.cs

```diff
@@ -0,0 +1,21 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue29661 : _IssuesUITest
+{
+	public override string Issue => "[iOS, Mac] StrokeDashArray Property not Rendering";
+
+	public Issue29661(TestDevice device)
+	: base(device)
+	{ }
+
+	[Test]
+	[Category(UITestCategories.Border)]
+	public void VerifyBorderWithStrokeDashArray()
+	{
+		App.WaitForElement("StrokeDashArrayLabel");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build.
![image](https://github.com/user-attachments/assets/180f9470-a25e-4eff-b2ef-c7ee8db3efe4)
Could you commit the images?

Reply:
@jsuarezruiz, I have added the snapshots for all platforms. Please let me know if any concerns